.markdownlint.json

```diff
@@ -5,6 +5,7 @@
   "list-marker-space": { "ol_multi": 2, "ol_single": 2 },
   "no-alt-text": false,
   "no-bare-urls": false,
+  "first-line-heading": false,
```

Comment:
Context: https://github.com/hashintel/hash/pull/159#discussion_r783849546

Looks like there is no way to disable a rule for a subset of files, so I turned it off everywhere. If the quality of our markdowns degrades because of that, we’ll rethink. TBH I don’t think this will happen.


`MD041` == `first-line-heading`